packages/react-native-sdk/src/utils/setClientDetails.ts

```diff
@@ -3,9 +3,11 @@ import {
   setDeviceInfo,
   setOSInfo,
   SfuModels,
+  setWebRTCInfo,
 } from '@stream-io/video-client';
 import { Platform } from 'react-native';
 import { version } from '../version';
+import RNWebRTCPackageJSON from '@stream-io/react-native-webrtc/package.json';
```

Comment:
Does this work? I think the SDK is doing self-importing. Perhaps change to `import package from '../../package.json'?
Or have this value injected as part of the build process?
- https://github.com/GetStream/stream-video-js/blob/main/packages/react-sdk/rollup.config.mjs#L32-L35
- https://github.com/GetStream/stream-video-js/blob/main/packages/react-sdk/index.ts#L17-L26